infra/modules/azure_app_service/tests/setup/main.tf

```
@@ -31,17 +31,17 @@ data "azurerm_subnet" "pep" {
   resource_group_name  = "${module.naming_convention.project}-network-rg-01"
 }
 
-resource "azurerm_resource_group" "rg" {
-  name     = "${module.naming_convention.prefix}-rg-app-${module.naming_convention.suffix}"
-  location = var.environment.location
+data "azurerm_resource_group" "rg" {
+  name = "${var.environment.prefix}-${var.environment.env_short}-itn-${var.environment.app_name}-rg-${module.naming_convention.suffix}"
```

It is probably better to avoid the dependency on the **app_name** `${var.environment.app_name}`, because the rg name is defined separately
```suggestion
  name = "${var.environment.prefix}-${var.environment.env_short}-itn-test-rg-${module.naming_convention.suffix}"

```

---

Done!